src/domain/events/utils.ts

```diff
@@ -33,9 +36,24 @@ export const getInitialValues = (
 ): EventSearchFormValues => {
   return {
     text: getTextFromDict(query, 'text') || '',
+    inLanguage: arrayQueryParameterToFormValue(
+      query.inLanguage
+    ) as EVENT_LANGUAGES[],
   };
 };
 
+export const arrayQueryParameterToFormValue = <T>(value?: T | T[]): T[] => {
```

Comment:
Helper function like this might have been implemented somewhere already?

Reply:
Maybe you could name this function little bit more generic way e.g. queryParameterToArray and move to src/utils folder? It could be easier to find